multicore_runtime/wsgi.py

```diff
@@ -43,6 +43,19 @@
 
 root_logger.setLevel(logging.INFO)
 
+# Enable Python Cloud Debugger if one is present. The debugger agent
+# communicates with the Cloud Debugger backend.
+#
+# The debugger has no effect on the application until snapshot is
+# requested.
+#
+# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
+try:
```

Comment:
On second thought, as long as we don't have the package as part of setup.py (and instead put it in the Dockerfile) and there's no reason for users to install it, we don't really need the check. But to answer your question, I am not familiar with that environment variable and it's not set inside the runtime itself to the best of my knowledge, so I assume it's set by the devappserver and therefore will be the same.